File: PHP/1768. Merge Strings Alternately.php
<?php

/**
 * @noinspection PhpIllegalPsrClassPathInspection
 * @noinspection PhpMultipleClassDeclarationsInspection
 * @noinspection PhpUnused
 */
class Solution
{
    public function mergeAlternately(string $word1, string $word2): string
    {
        $result = '';
        $length = max(strlen($word1), strlen($word2));

        for ($i = 0; $i < $length; $i++) {
            $result .= (($word1[$i] ?? '').($word2[$i] ?? ''));
        }

        return $result;
    }
}

File: Python/1768. Merge Strings Alternately.py
# noinspection PyMethodMayBeStatic
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        result = ''
        length = min(len(word1), len(word2))

        for i in range(length):
            result += word1[i] + word2[i]

        i += 1
        return result + word1[i:] + word2[i:]

